src/OpenTelemetry.Exporter.Geneva/TLDExporter/JsonSerializer.cs

```diff
@@ -104,6 +104,22 @@ public static int SerializeString(byte[] buffer, int cursor, string value)
         return cursor;
     }
 
+#if NET6_0_OR_GREATER
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static int SerializeString(byte[] buffer, int cursor, ReadOnlySpan<char> value)
+    {
+        if (value == null)
```

Comment:
`ReadOnlySpan` is a struct so you probably want to something like check `value.IsEmpty` here?

Reply:
Fixed it.